grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -199,4 +200,28 @@ class NewConsentGroupController extends AuthenticatedController {
         }
         render(response)
     }
+
+
+    /**
+     * This action approve the link between a project and the consent.
+     *
+     */
+    def approveLink () {
+        try {
+            boolean isUpdated = queryService.updateCollectionLinkStatus(params.consentKey, params.projectKey, IssueStatus.Approved.name)
+            if (!isUpdated) {
+              response.status = 400
+              render([message: 'Error updating collection links, please check specified parameters.'] as JSON)
+            } else {
+              response.status = 200
+              render([message: 'Collection links are been updated.'] as JSON)
```

Comment:
Similar concern with respect to returning information to the user. A json list of created links would be very appropriate.